snorkel/analysis/error_analysis.py

```diff
@@ -6,33 +6,32 @@
 from .utils import to_int_label_array
 
 
-def error_buckets(
-    golds: np.ndarray, preds: np.ndarray
-) -> Dict[Tuple[int, int], List[int]]:
-    """Return data point indices bucketed by gold label/pred label combination.
+def label_buckets(*y: np.ndarray) -> Dict[Tuple[int, int], np.ndarray]:
```

Comment:
`label buckets` can be read as `label the buckets`, which we want to avoid, especially with a lot of classes in this library that do actually label stuff and have label in their name. How about `get_label_buckets` or `get_label_groups`?